### PR TITLE
upgrade sylius/theme-bundle, upgrade project to Symfony2.8 again

### PR DESCRIPTION
**Pull request by ahilles107**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | no
| Fixed tickets | no
| License       | GPLv3

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -42,9 +42,19 @@
             "**/Tests/", "**/spec/"
         ]
     },
+    "repositories": [
```
Comment (takeit) [inline composer.json]: this should be removed

File: src/SWP/Bundle/MultiTenancyBundle/EventListener/TenantableListener.php [hunk 2]
```diff
@@ -53,8 +53,8 @@ public function __construct(EntityManagerInterface $entityManager, TenantContext
     public function onKernelRequest()
     {
         $tenant = $this->tenantContext->getTenant();
-        $tenantId = $tenant->getId();
-        if ($tenantId) {
+        if ($tenant && $tenant->getId()) {
```
Comment (takeit) [inline src/SWP/Bundle/MultiTenancyBundle/EventListener/TenantableListener.php]: :+1:

File: composer.json [hunk 3]
```diff
@@ -61,7 +71,7 @@
         "emanueleminotto/twig-cache-bundle": "~1.0",
         "incenteev/composer-parameter-handler": "~2.0",
         "friendsofsymfony/http-cache-bundle": "~1.0",
-        "friendsofsymfony/rest-bundle": "2.0.x-dev",
+        "friendsofsymfony/rest-bundle": "2.0.x-dev as 1.5",
```
Comment (takeit) [inline composer.json]: is this for SyliusResourceBundle?

File: composer.json (hunk 1 above)
Comment (ahilles107) [inline composer.json]: Yep

Comment (ahilles107) [inline composer.json]: For now it need to stay. We don't control sylius repos, and without it we can't use aliases.

File: composer.json (hunk 3 above)
Comment (ahilles107) [inline composer.json]: yep, it need to stay like that as long as Sylius will support 2.0.x-dev this same with "sylius/resource-bundle"

**Pull request merged**